javascript-nodejs-stream/receive.js

```diff
@@ -0,0 +1,37 @@
+const rabbit = require("rabbitmq-stream-js-client")
+
+const sleep = (ms) => new Promise((r) => setTimeout(r, ms))
+
+async function main() {
+    const streamName = "test-queue-stream"
+
+    console.log("Connecting...");
+    const client = await rabbit.connect({
+        hostname: "localhost",
+        port: 5552,
+        username: "guest",
+        password: "guest",
+        vhost: "/",
+    })
+
+    console.log("Making sure the stream exists...");
+    const streamSizeRetention = 5 * 1e9
+    await client.createStream({ stream: streamName, arguments: { "max-length-bytes": streamSizeRetention } });
+
+    console.log("Declaring the consumer with offset...");
+    await client.declareConsumer({ stream: streamName, offset: rabbit.Offset.first() }, (message) => {
+        console.log(`Received message ${message.content.toString()}`)
+    })
+
+    await sleep(2000)
```

Comment:
@icappello we'd need to change the consumer and make it like the others.  
so it should consume continuously and not stopping after two seconds